### PR TITLE
Add Diagram it mentioning to docs

### PR DESCRIPTION
**Pull request by marcosvst**
Related to this comment https://github.com/yuzutech/kroki/issues/867#issuecomment-915788485

### CODE REVIEW

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -18,3 +19,6 @@ Thanks to {uri-exoscale}[Exoscale], which has kindly offered two servers, we can
 However, some parts of the infrastructure are still host on other providers, so *we are still looking for sponsors to cover all the expenses*.
 +
 If you are interested, please 👋 mailto:hello@kroki.io[contact us].
+
+The public instance of Kroki does not do any remote call for performance/security reasons. Also we want to keep the API as simple as possible, so it takes a text diagram as input and returns an image.
+If you want to generate diagrams directly from a public source file, check this other project named {uri-diagramit}[Diagram it] that was built upon Kroki and allow you to generate the diagram only by providing the diagram's public source file path.
```
Comment (Mogztter) [inline docs/modules/ROOT/pages/index.adoc:24]: I think it might be better to link to: https://diagramit.herokuapp.com/ no?

**Comment by Mogztter** [comment]
Thanks @marcosvst!
The content is great but I'm not convinced that the home page of the documentation is the right place. I think we should create a new page named "Integration" or "Third Party Tools" with a section about "Diagram it".

We could also use this new page to talk about other projects that are related to Kroki (or that uses Kroki).

What do you think?

**Comment by marcosvst** [comment]
> Thanks @marcosvst!
> The content is great but I'm not convinced that the home page of the documentation is the right place. I think we should create a new page named "Integration" or "Third Party Tools" with a section about "Diagram it".
> 
> We could also use this new page to talk about other projects that are related to Kroki (or that uses Kroki).
> 
> What do you think?

You're right, that makes more sense.

Thanks for updating the PR.

**Comment by Mogztter** [comment]
No worries, thanks for reviewing.

**Comment by Mogztter** [comment]
@marcosvst For reference, I get an "Internal Server Error" when using the following URL: https://diagramit.herokuapp.com/v1/plantuml/generate?source=https://raw.githubusercontent.com/yuzutech/kroki/main/tests/smoke/diagrams/architecture.puml

**Comment by marcosvst** [comment]
> @marcosvst For reference, I get an "Internal Server Error" when using the following URL: https://diagramit.herokuapp.com/v1/plantuml/generate?source=https://raw.githubusercontent.com/yuzutech/kroki/main/tests/smoke/diagrams/architecture.puml
> 
> I also noticed that the dropdown only has an handful of options but if you type the name of the diagram library in the URL it's working as expected.
> For instance, Vega is not in the dropdown but you can use:
> 
> https://diagramit.herokuapp.com/v1/vega/generate?source=https://github.com/yuzutech/kroki/blob/main/tests/smoke/diagrams/bar-chart.vega

@Mogztter thanks for spotting that, it was a silly bug but that is fixed now.

Re: dropdown only having a few options - that was expected, the backend is prepared to handle all diagram types Kroki supports, I didn't add them all to the generator webpage yet because of lack of time.

**Comment by Mogztter** [comment]
> Re: dropdown only having a few options - that was expected, the backend is prepared to handle all diagram types Kroki supports

👍🏻 

> I didn't add them all to the generator webpage yet because of lack of time.

No worries, I can relate 😉

**Pull request merged**